### DISCUSSION TITLE
Ruby 3 upgrades?

### DISCUSSION THREAD

**Discussion by yifeiwu** [aws/aws-sdk-ruby#2787] [Q&A]
Does anyone know what the Ruby version dependency is for the SDK? Also am interested in knowing if the lambda runtimes are going to get updated. Seems like the only one available is for ruby 2.7.

**Comment by mullermp** [comment]
We require any Ruby version greater than 2.3. As for lambda, I've requested newer Ruby versions and I think it's being worked on though I don't know the status.

**Reply by yifeiwu** [reply]
Thanks, do you know if there's a better place to ask this question, i.e. so the team working on it might have an answer?

**Reply by mullermp** [reply]
Externally, customer support would probably be the best bet. Unofficially, it looks like it may be soon (TM). I've forwarded this issue to the Lambda team who may respond.
